docs/contributing/review-process.md

```diff
@@ -0,0 +1,172 @@
+# Pull request review process
+
+Pull requests submitted to Zulip go through a rigorous review process, which is
+designed to ensure that we are building a high-quality product with a
+maintainable code base. This page describes the stages of review your pull
+request may go through, and offers guidance on how you can help keep your pull
+request moving along.
+
+In the Zulip server/web app repository
+([`zulip/zulip`](https://github.com/zulip/zulip/)), we use GitHub labels to help
+everyone understand where a pull request is in the review process. These labels
+are noted below. Each label is removed by the reviewer for that stage when they
+have no more feedback on the PR, and consider it ready for the next stage of
```

Comment:
I'd remove the comma before the "and" here. It reads to me like it's a second thought after "Each label" vs referring to the reviewer part.